libs/s25main/mapGenerator/Random.h

```diff
@@ -22,14 +22,19 @@
 #include "mapGenerator/MapSettings.h"
```

Comment:
Another hint: Check what includes you really need. Here you need none. Everything can be forward declared and included in the cpp file. Well if you really meant to keep a RandomUtility  instance then you need 1 include.

In general: Double check all headers for includes that you may not need and replace them by forward declarations wherever possible. Keeps compiletimes and recompilations down.

Reply:
Removed lots of includes which are not required.